composer.json

```diff
@@ -1,12 +1,12 @@
 {
-    "name": "bkrukowski/transparent-email",
+    "name": "gridnevalex/transparent-email",
```

Comment:
? :)

Reply:
Please ignore this commit) I've just added this for testing purposes (for installing my modified version via composer)

Reply:
I'm sorry for that. I will return it back in the new PR.